### PR TITLE
OKTA-538109 Add user attribute mapping note for group functions

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add user profile attribute mapping note for group functions
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-538109](https://oktainc.atlassian.net/browse/OKTA-538109)

### CODE REVIEW

**Review by josephwei-okta — APPROVED** [review]
lgtm

**Review by williamyang-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md [hunk 1]
```diff
@@ -188,6 +188,8 @@ Group functions return either an array of groups or **True** or **False**.
 
 > **Note:** The `isMemberOfGroupName`, `isMemberOfGroup`, `isMemberOfAnyGroup`, `isMemberOfGroupNameStartsWith`, `isMemberOfGroupNameContains`, `isMemberOfGroupNameRegex` group functions are designed to retrieve only an Okta user's group memberships. Don't use them to retrieve an app user's group memberships.
 
+> **Note:** When EL group functions (such as `isMemberOfGroup` or `isMemberOfGroupName`) are used for app assignments, app user profile attributes aren’t updated or reapplied when the user’s group membership changes. Okta updates app user profile attributes only when an app is assigned to a user or when mappings are applied.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md:191]:
```suggestion
> **Note:** When EL group functions (such as `isMemberOfGroup` or `isMemberOfGroupName`) are used for app assignments, app user profile attributes aren’t updated or reapplied when the user’s group membership changes. Okta only updates app user profile attributes when an app is assigned to a user or when mappings are applied.
```

**Review by susanharper-okta — APPROVED** [review]
one nit - LGTM

**Pull request merged**